### PR TITLE
Added information about NODE_PATH issue

### PR DESCRIPTION
**Pull request by ieugen**
Signed-off-by: Ioan Eugen Stan <ieugen@apache.org>

### CODE REVIEW

File: bin/grommet.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ function getGrommetPath() {
   });
 
   if (!grommetPath) {
-    console.log('Could not find Grommet!');
+    console.log('Could not find Grommet! Check NODE_PATH is set.');
```
Comment (alansouzati) [inline bin/grommet.js]: Can we say `Please make sure you've set NODE_PATH to your version of NPM` ?

File: docs/src/develop/GetStarted.htm [hunk 2]
```diff
@@ -49,6 +49,8 @@
             </li>
             <li>
             	<p>Use the Grommet generator to bootstrap your new app</p>
+              <p>If you get 'Could not find Grommit' error make sure you set the
```
Comment (alansouzati) [inline docs/src/develop/GetStarted.htm]: Can you make sure that the spacing is write in this file ? I think we have one extra unnecessary space.

**Comment by alansouzati** [comment]
Thanks for your pull request. I've sent some minor feedbacks, hopefully you can change that.

File: docs/src/develop/GetStarted.htm (hunk 2 above)
Comment (ieugen) [inline docs/src/develop/GetStarted.htm]: It doesn't show on my machine. I've checked with both VIM and atom. I have editorconfig plugin. Can't figure it out.

Comment (alansouzati) [inline docs/src/develop/GetStarted.htm]: you're right. it seems a formatting issue with github.

**Comment by alansouzati** [comment]
Thanks for your contribution @ieugen

**Pull request merged**